<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SKS_GUI_RefactorClass</class>
 <widget class="QMainWindow" name="SKS_GUI_RefactorClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Serial Keyboard Server</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: #303030;
	color: rgb(66, 221, 255);
	font: 8pt &quot;Consolas&quot;;
}

QMainWindow{
	background-image : url(:/SKS_GUI_Refactor/Plan de travail 1.png);
}

QCheckBox, QRadioButton {
	border: none;
}

QRadioButton::indicator, QCheckBox::indicator {
	width: 13px;
	height: 13px;
}

QRadioButton::indicator::unchecked, QCheckBox::indicator::unchecked {
	border: 1px solid #5A5A5A;
	background: none;
}

QRadioButton::indicator:unchecked:hover, QCheckBox::indicator:unchecked:hover {
	border: 1px solid rgb(66, 221, 255);
}

QRadioButton::indicator::checked, QCheckBox::indicator::checked {
	border: 1px solid #5A5A5A;
	background: rgb(16, 171, 205);
}

QRadioButton::indicator:checked:hover, QCheckBox::indicator:checked:hover {
	border: 1px solid rgb(66, 221, 255);
	background: rgb(46, 201, 235);
}

QProgressBar {
	 border: 1px solid rgb(66, 221, 255);
     border-radius: 5px;
     background-color:  #252525;
}

QProgressBar::chunk {
	background-color: rgb(66, 221, 255);
 	width: 15px;
    margin: 0.5px;
}


QMenu::item::selected,QMenuBar::item:selected{
	background-color: #404040;
}

QMenu::item::selected,QMenuBar::item:pressed{
	background-color: #505050;
}

QLineEdit{
border: 1px solid #5A5A5A;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelPort">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>382</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial Port :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputPort">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>381</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonStart">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>379</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonStop">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>379</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>751</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>21</width>
      <height>181</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>21</width>
      <height>181</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>21</width>
      <height>161</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>21</width>
      <height>161</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>761</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="LayoutTerm"/>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>383</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>567</x>
      <y>380</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Server stopped</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Name1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Name2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Name3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Name4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Name5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Name6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="KeyGrab5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Appuyer pour configurer</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Hold5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>162</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Hold</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="KeyGrab6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Appuyer pour configurer</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Hold6">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>162</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hold</string>
    </property>
   </widget>
   <widget class="QPushButton" name="KeyGrab3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Appuyer pour configurer</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="Hold3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>72</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hold</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Hold2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>72</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hold</string>
    </property>
   </widget>
   <widget class="QPushButton" name="KeyGrab2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Appuyer pour configurer</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="KeyGrab1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Appuyer pour configurer</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="Hold1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>72</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hold</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="KeyGrab4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Appuyer pour configurer</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Hold4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>162</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hold</string>
    </property>
   </widget>
   <zorder>KeyGrab5</zorder>
   <zorder>Hold5</zorder>
   <zorder>KeyGrab6</zorder>
   <zorder>Hold6</zorder>
   <zorder>KeyGrab3</zorder>
   <zorder>Hold3</zorder>
   <zorder>Hold2</zorder>
   <zorder>KeyGrab2</zorder>
   <zorder>KeyGrab1</zorder>
   <zorder>Hold1</zorder>
   <zorder>KeyGrab4</zorder>
   <zorder>Hold4</zorder>
   <zorder>labelPort</zorder>
   <zorder>InputPort</zorder>
   <zorder>ButtonStart</zorder>
   <zorder>ButtonStop</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
   <zorder>line_2</zorder>
   <zorder>line_5</zorder>
   <zorder>line_6</zorder>
   <zorder>line</zorder>
   <zorder>line_7</zorder>
   <zorder>progressBar</zorder>
   <zorder>labelStatus</zorder>
   <zorder>Name1</zorder>
   <zorder>Name2</zorder>
   <zorder>Name3</zorder>
   <zorder>Name4</zorder>
   <zorder>Name5</zorder>
   <zorder>Name6</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_configuration"/>
    <addaction name="actionSave_configuration"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionLoad_configuration">
   <property name="text">
    <string>Load configuration</string>
   </property>
  </action>
  <action name="actionSave_configuration">
   <property name="text">
    <string>Save configuration</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SKS_GUI_Refactor.qrc"/>
 </resources>
 <connections/>
</ui>
